zebra-utils/README.md

```diff
@@ -157,8 +157,8 @@ Response diff (between zcashd port and port 28232):
 +++ /run/user/1000/tmp.g9CJecu2Wo/zcashd-main-1649797-getinfo.json      2022-04-29 14:08:46.769240315 +1000
 @@ -1,4 +1,16 @@
  {
--  "build": "1.0.0-beta.8+54.ge83e93a",
--  "subversion": "/Zebra:1.0.0-beta.8/"
+-  "build": "1.0.0-rc.9+54.ge83e93a",
+-  "subversion": "/Zebra:1.0.0-rc.9/"
```

Comment:
I think we might want to leave this as it was because it is the output of a command executed at `2022-04-29 14:08:46`. If we want to change it, it should be better to get a full new output and paste it here.

Reply:
backed out in 88f3120bca4732d3727363e4ad75de0eb19856e5